mirrord/layer/src/load.rs

```diff
@@ -38,6 +39,17 @@ static BUILD_TOOL_PROCESSES: LazyLock<HashSet<&str>> = LazyLock::new(|| {
     ])
 });
 
+static COMBINED_BUILD_TOOLS: LazyLock<HashSet<String>> = LazyLock::new(|| {
+    let mut combined = BUILD_TOOL_PROCESSES
+        .iter()
+        .map(|&s| s.to_string())
+        .collect::<HashSet<String>>();
+    if let Ok(extra_tools) = env::var("MIRRORD_SKIP_EXTRA_BUILD_TOOLS") {
+        combined.extend(extra_tools.split(';').map(|s| s.trim().to_string()));
```

Comment:
When you make the changes I'm suggesting, you can use `deref().map([_])::to_vec()` on the config (since it's a `VecOrSingle`) and not have to manually `split` it.

Reply:
Neat! Thanks alex.